.github/workflows/ci.yml

```diff
@@ -223,3 +225,44 @@ jobs:
         with:
           path: perf-measures/type-check/previous-result.txt
           key: type-check-perf-previous-result-${{ github.sha }}
+
+  perf-measures-bundle-check-on-pr:
+    name: 'Bundle Check on PR'
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
```

Comment:
q: Does the bundle size change every time?
Or does it change only when a specific file is changed?

Should I integrate workflow jobs when importing into https://github.com/honojs/hono/pull/3491? Should we keep them separate? I am pondering.

Reply:
> q: Does the bundle size change every time?
> Or does it change only when a specific file is changed?

`it change only when a specific file is changed` is right.

Only when a core file is changed.

> Should I integrate workflow jobs when importing into https://github.com/honojs/hono/pull/3491? Should we keep them separate? I am pondering.  

I have limited knowledge about it, sorry.

Reply:
I would like to keep this as a TODO/NOTE as it seems a bit difficult to implement.

https://stackoverflow.com/questions/70708306/github-actions-run-step-job-in-a-workflow-if-changes-happen-in-specific-folde